README.md

```diff
@@ -568,10 +610,43 @@ interface Metric {
   // by the Navigation Timing API (or `undefined` if the browser doesn't
   // support that API). For pages that are restored from the bfcache, this
   // value will be 'back_forward_cache'.
-  navigationType:  NavigationType | 'back_forward_cache' | undefined;
+  navigationType:  NavigationType | 'back_forward_cache' | 'prerender' | undefined;
 }
 ```
 
+Metric-specific subclasses:
+
+- [`CLSMetric`](/src/types/cls.ts#:~:text=interface%20CLSMetric)
+- [`FCPMetric`](/src/types/cls.ts#:~:text=interface%20FCPMetric)
+- [`FIDMetric`](/src/types/cls.ts#:~:text=interface%20FIDMetric)
+- [`INPMetric`](/src/types/cls.ts#:~:text=interface%20INPMetric)
+- [`LCPMetric`](/src/types/cls.ts#:~:text=interface%20LCPMetric)
+- [`TTFBMetric`](/src/types/cls.ts#:~:text=interface%20TTFBMetric)
+
+#### `MetricWithAttribution`
+
+See the [attribution build](#attribution-build) section for details on how to use this feature.
```

Comment:
This section doesn't really show how to use these - it just documents the API.

Could we update the Universal Analytics/GTM section on lines 338 - 467 to include these?
Especially as it currently has the following:

```
    // OPTIONAL: any additional params or debug info here.
    // See: https://web.dev/debug-web-vitals-in-the-field/
    // metric_rating: 'good' | 'ni' | 'poor',
    // debug_info: '...',
    // ...
```

Reply:
Yep, will add.